index.js

```diff
@@ -31,15 +31,23 @@ function defaultArrayMerge(target, source, optionsArgument) {
 }
 
 function mergeObject(target, source, optionsArgument) {
+    var clone = optionsArgument && optionsArgument.hasOwnProperty('clone') 
```

Comment:
`var clone = optionsArgument && optionsArgument.close === true` is probably safe enough, since we're fine with defaulting to `false`.